### ISSUE TITLE
A lot of keywords not checked for word boundaries

### ISSUE THREAD

**Issue by belyaev-mikhail** [vkostyukov/kotlin-sublime-package#18]
Anything starting with a "class" or "object" or maybe other keywords now triggers highlighting the line as a class header. If the same line somehow contains a '{' symbol, the rest of the file is not highlighted.

```kotlin
val x = className + "{"
val b = foo()
val a = bar()
```

```kotlin
val x = objectValue; val z = "{"
val b = foo()
val a = bar()
```

**Comment by belyaev-mikhail** [comment]
#6 is a specific case of the same issue, it seems

```kotlin
fun foo() {
    val x  = Bar::class
    val y = hello
}
```

**Comment by belyaev-mikhail** [comment]
As is #4 

```kotlin
val x = valueOfMe + "=4asdasdasdas"
val z = 5
```

**Comment by belyaev-mikhail** [comment]
As is #8 

(see "value" and "=8" on the same line there)

```kotlin
    for ((key, value) in BlockOre.inventoryVariants) {
        val parts = value.split("=")
        val name = parts[0] + parts[1].toLowerCase().capitalize()
        if (key == 2) {
            OreDictionary.registerOre("${name}_Mgc", ItemStack(BlockOre, 1, key))
        } else {
            OreDictionary.registerOre(name, ItemStack(BlockOre, 1, key))
        }
    }
```

**Comment by belyaev-mikhail** [comment]
Same goes for `fun`.

```kotlin
val x = funToPlayWith + "(){"
val b = "Hello"
val x = "}"
val z = foo()
```

**Comment by belyaev-mikhail** [comment]
Also #9

**Comment by vkostyukov** [comment]
Should be fixed by #20 (I hope). Feel free to re-open if not.
